pages/Verification.md

```diff
@@ -22,7 +22,7 @@ To get up and running with the UD CIS Discord server, you'll need to complete a
 If you have any problems with the above steps, please contact one of the server's maintainers at
 **[joshlyon@udel.edu][32] | [briross@udel.edu][34]**
```

Comment:
Simon and I </3

Reply:
i swear every week we find another thing we forgot to add you guys too